docs/pages/includes/database-access/cloudsql_service_account_key.mdx

```diff
@@ -11,3 +11,15 @@ Make sure to choose JSON format:
 
 Save the file. Your Teleport Database Service will need to use it as GCP
 application credentials file.
+
+<Admonition type="warning">
+A service account key can be a security risk - we only describe using a key in
+this guide for simplicity.
+We do not recommend using service account keys in production.
+If the Teleport Database Service is running on Google Cloud then the recommended
+approach is to attach a service account to its Compute Engine instance.
```

Comment:
FYI. I've tried out GCP [Workload Identity Federation](https://cloud.google.com/iam/docs/workload-identity-federation) recently as a customer wants to use that for GCP PostgreSQL backend. I think it's a good secure option for non-Google cloud environment if you want to mention it here.

Reply:
added this:
>For non-GCE deployments, we recommend using
[workload identity](https://cloud.google.com/iam/docs/workload-identity-federation).